packages/react/src/patterns/sub-patterns/Quote/README.md

```diff
@@ -0,0 +1,103 @@
+# Quote
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+@import '@carbon/ibmdotcom-styles/scss/patterns/sub-patterns/quote';
+```
+
+> 💡 Only import fonts once per usage. Don't forget to import the Quote styles
+> from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
+
+```javascript
+import { Desktop } from '@carbon/pictograms-react';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Quote } from '@carbon/ibmdotcom-react';
+import 'yourapplication.scss';
+
+function App() {
+  const copy = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.';
+
+  const cta = {
+    type: 'local',
+    href: 'https://www.example.com',
+    copy: 'Lorem ipsum dolor',
+  };
+
+  const source = {
+    heading: 'lorem ipsum',
+    copy: 'dolor sit amet',
+  };
+
+  return <Quote copy={copy} cta={cta} source={source} />;
+}
+
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+Add the following line on your `.env` file at the root of your project,
+[see more details](https://github.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/styles#usage)
+
+```
+  SASS_PATH=node_modules:src
+```
+
+> 💡 Don't forget to import the quote styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
+
+## Props
+
+| Name      | Required | Data Type | Default Value  | Description                                                                                                                                             |
+| --------- | -------- | --------- | -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `copy`    | YES      | String    | null           | Main Quote                                                                                                                                              |
+| `types`   | NO       | String    | `doubleCurved` | See `Types` bellow.                                                                                                                                     |
```

Comment:
minor thing, should it be `type` and not `types`?

Reply:
i've fixed that on the latest commit, good catch